util/NarrativeGenerator.xsl

```diff
@@ -372,8 +367,8 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
                         </xsl:for-each>
                         <xsl:call-template name="doResourceContext">
                             <xsl:with-param name="in" select="f:context | f:extension[@url = 'http://nictiz.nl/fhir/StructureDefinition/extension-context-nl-core-episodeofcare']/f:valueReference"/>
-                            <xsl:with-param name="textLang" select="$textLang"/>
-                        </xsl:call-template>
+                                        <xsl:with-param name="textLang" select="$textLang"/>
+                                    </xsl:call-template>
```

Comment:
Vreemde whitespace-aanpassing hier en in de rest van het bestand. Maar ligt misschien aan de rendering van GitHub

Reply:
Zal wel gebeurd zijn tijdens het handmatige oplossen van het merge conflict. Ik heb het iig aangepast.